src/agent/at-driver.js

```diff
@@ -53,10 +59,11 @@ export class ATDriver {
   async sendKeys(...keys) {
     for (const chord of ATKey.sequence(...keys)) {
       for (const { key } of chord) {
-        await this._send({ type: 'command', name: 'pressKey', params: [key] });
-      }
-      for (const { key } of Array.from(chord).reverse()) {
-        await this._send({ type: 'command', name: 'releaseKey', params: [key] });
+        await this._send({
+          type: 'command',
+          method: 'interaction.pressKeys',
+          params: { keys: chord.keys.map(({ mapped }) => mapped) },
```

Comment:
This inline map was the "quick and dirty" way to get it working, Thinking about moving this onto `chord.toAtDriverKeysArray()`

Reply:
Yeah, in my initial comments above I added that I want to pull out the "quick map" into something like `chord.toAtDriverKeysArray()` with a better structure

Reply:
I made this into a method on chord (and got rid of the xtra for loop)